src/main/java/network/brightspots/rcv/Logger.java

```diff
@@ -83,7 +84,7 @@ class Logger {
 
   static void setup() {
     logger = java.util.logging.Logger.getLogger("");
-    logger.setLevel(Level.FINE);
+    logger.setLevel(AUDIT_LEVEL);
```

Comment:
What about line 132. Does it make sense that it should still do: `tabulationHandler.setLevel(Level.FINE)`?

Reply:
No it does not! Good catch as well. The whole point was to reduce the audit log spam from FINEs :) fixed